wagtail/models/__init__.py

```diff
@@ -3468,12 +3627,14 @@ def locked_for_user(self, obj, user):
 
     def user_can_lock(self, obj, user):
         """Returns True if a user who would not normally be able to lock the object should be able to if the object is currently on this task.
-        Note that returning False does not remove permissions from users who would otherwise have them."""
+        Note that returning False does not remove permissions from users who would otherwise have them.
```

Comment:
I'm sorry about that. I'm not sure why this happened. I only used Ruff and Black for formatting because the checks usually fail when it's not formatted properly. Could it be because of that?

Reply:
I'm sorry about that. I'm not sure why this happened. I only used Ruff and Black for formatting because the checks usually fail when it's not formatted properly. Could it be because of that?

Reply:
The black formatter is reformatting the file and causing this. But if I don't reformat the file won't the checks fail?